printer_mods/Eifel-Joe/V0.1_Waveshare_4.3_inch_TFT_Mount/Readme.md

```diff
@@ -0,0 +1,24 @@
+# Voron v0.1 4.3" Touchscreen Mount
+
+This mod mounts a [Waveshare 4.3inch Capacitive Touch Display](https://www.waveshare.com/4.3inch-dsi-lcd.htm) purchased from [Amazon](https://www.amazon.de/gp/product/B08CH24YYD/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&psc=1).  The screen is oriented at a 55 degree angle
+
+<div align="center">
+    <img src="Image/Waveshare43_Mount.png" height="400">
+</div>

```

Comment:
No HTML in Markdown flies please.

Reply:
-Spaces in filenames are removed
-STEP format for CAD added
-README.md new. Don't know what happend last time
-Photo above added to README
-Removed HTML-Code